src/Confluent.Kafka/ProduceException.cs

```diff
@@ -31,13 +31,32 @@ public class ProduceException<TKey, TValue> : KafkaException
         /// <param name="error"> 
         ///     The error associated with the delivery report.
         /// </param>
-        /// <param name="deliveryReport">
+        /// <param name="deliveryResult">
```

Comment:
note: a `DeliveryResult` is a `DeliveryReport` without the error, useful in several contexts.